<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QA-Mickael-Lap" name="com.automation.main.search.TC18881ValidateTheSourceTypeAsChapterKeywordInSearchFieldOnTheAllCoursesLevel" tests="1" failures="1" timestamp="18 Oct 2016 00:42:54 GMT" time="54.989" errors="0">
  <testcase name="test18881" time="54.989" classname="com.automation.main.search.TC18881ValidateTheSourceTypeAsChapterKeywordInSearchFieldOnTheAllCoursesLevel">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
	at com.automation.main.page_helpers.SearchPage.verfiyBreadcrumbStructureDisplayedAsCoursesXResultsFound(SearchPage.java:515)
	at com.automation.main.search.TC18881ValidateTheSourceTypeAsChapterKeywordInSearchFieldOnTheAllCoursesLevel.test18881(TC18881ValidateTheSourceTypeAsChapterKeywordInSearchFieldOnTheAllCoursesLevel.java:204)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- test18881 -->
</testsuite> <!-- com.automation.main.search.TC18881ValidateTheSourceTypeAsChapterKeywordInSearchFieldOnTheAllCoursesLevel -->
